wp-media/wp-rocket
# Closes #2842 delete minified files on theme switch

## Description

Added a new AdminSubscriber class to to listen for `switch_theme `event and clean cache/min with `rocket_clean_minify()`

Fixes #2842 

## Type of change


## Is the solution different from the one proposed during the grooming?

Solution is in no way different from the proposed one.

## How Has This Been Tested?

- Enable Minify & Merge Options for both CSS & JS
- Activate a new theme
- Load new site in incognito with Ctrl+f5(windows) for force refresh
- check the wp-content/cache/min folder to see minified files generated
- Activate another theme
- check to see that previous minified files have been removed.


# Checklist:

Please delete the options that are not relevant.
